src/test/unit/specific_feature_tests/image-resize-handle.js

```diff
@@ -127,3 +129,17 @@ test("resize handle with no image is hidden on `mousemove`", function () {
         }
     });
 });
+
+test("resize handle detached before adding history", function () {
+    expect(1);
+    var wymeditor = jQuery.wymeditors(0);
+    var spy = sinon.spy(wymeditor.ih, '_detachResizeHandle');
+
+    wymeditor.undoRedo.reset();
+    wymeditor.html('<p><img src="' + IMG_SRC + '" /></p>');
+    wymeditor.$body().find('img')
+        .mousemove();
+    wymeditor.undoRedo._add();
+
+    strictEqual(spy.calledOnce, true);
```

Comment:
You're right, @michaelgzoller, it is not the same thing.
Also, it would be better to test that `_detachResizeHandle` is called before `_add` is called. And I don't do that, either. This seems to be the correct amount of love this old editor should get.